FrontEnd/src/components/ProfilePage/FormComponents/GeneralInfo.js

```diff
@@ -140,8 +146,25 @@ const GeneralInfo = (props) => {
     };
 
     const onUpdateField = e => {
+        const { value: fieldValue, name: fieldName } = e.target;
+        const symbolCount = (fieldValue.replace(/[\s`]/g,'').trim()).length;
```

Comment:
Regex replacement `fieldValue.replace(/[\s]/g,'')` removes spaces and backticks. The `trim()` might be redundant because all leading and trailing whitespace would already be removed by the regex

Reply:
Fixed